code/drasil-example/Drasil/Projectile/ProblemDescription.txt

```diff
@@ -0,0 +1,44 @@
+﻿Physical System Description
+
+Elements:
+	1) The object
+	2) The ground
+
+Assumptions
+1) The projectile motion is in 2D.
+2) The object is a point mass.
+3) The object's path starts and ends at ground level.
+4) The effects of drag, wind, and the Earth's curvature are ignored.
+5) The flight is free; there are no collisions during the trajectory of the object.
+6) The positive x-direction is the direction from start to the end of the object's path.
+7) The positive y-direction is the direction away from the ground.
+
+Goal Statments
+1) Check if the final position from user input is reached by the object with initial conditions.
```

Comment:
From the physics point of view, I think the goal is simply to calculate the landing position of the projectile.  You might have requirements related to how you convey this information to the user, but everything is based off of the goal of finding the final position.  If we want to reuse this ballistics trajectory, we are better off not revealing that we are using it as a "game" in the modelling portion.  In this way we can use the same goals for a different game, or simulation, simply by changing the requirements.